src/test/java/edu/ucla/library/iiif/auth/delegate/CantaloupeAuthDelegateIT.java

```diff
@@ -132,280 +174,474 @@
 
-    /**
-     * Tests that the HTTP response to a properly authorized request for an all-or-nothing access info.json, via Image
-     * API 3, indicates full access.
-     *
-     * @throws IOException If there is trouble sending the HTTP request(s) or getting the expected info.json response
-     * @throws InterruptedException If there is trouble sending the HTTP request(s)
-     */
-    @Test
-    public final void testFullAccessResponseAllOrNothingWithTokenV3() throws IOException, InterruptedException {
-        final HttpResponse<String> response = sendImageInfoRequest(ALL_OR_NOTHING_ACCESS_IMAGE, SINAI_ACCESS_TOKEN, 3);
-        final String expectedResponse =
-                getExpectedDescriptionResource(ALL_OR_NOTHING_ACCESS_IMAGE, FULL_ACCESS_RESPONSE_TEMPLATE_V3, 3);
-
-        Assert.assertEquals(HTTP.OK, response.statusCode());
-        TestUtils.assertEquals(expectedResponse, response.body());
+            Assert.assertEquals(HTTP.OK, response.statusCode());
+            Assert.assertTrue(TestUtils.responseHasContentType(response, MediaType.APPLICATION_JSON,
+                    MediaType.APPLICATION_LD_PLUS_JSON));
+            TestUtils.assertEquals(expectedResponse, response.body());
+        }
+
+        @Override
+        public final void testDegradedAccessResponseTieredUnauthorized() throws IOException, InterruptedException {
+            final HttpResponse<String> response = sendImageInfoRequest(TIERED_ACCESS_IMAGE, null, 2);
+            final String expectedResponse =
+                    getExpectedImageInfo(TIERED_ACCESS_IMAGE_DEGRADED_VALID, DEGRADED_ACCESS_RESPONSE_TEMPLATE_V2, 2);
+
+            Assert.assertEquals(HTTP.OK, response.statusCode());
+            Assert.assertTrue(TestUtils.responseHasContentType(response, MediaType.APPLICATION_JSON,
+                    MediaType.APPLICATION_LD_PLUS_JSON));
+            TestUtils.assertEquals(expectedResponse, response.body());
+        }
+
+        @Override
+        public final void testErrorResponseTieredDisallowedScale() throws IOException, InterruptedException {
+            final HttpResponse<String> response =
+                    sendImageInfoRequest(TIERED_ACCESS_IMAGE_DEGRADED_UNAVAILABLE, null, 2);
+
+            Assert.assertEquals(HTTP.FORBIDDEN, response.statusCode());
+            Assert.assertFalse(TestUtils.responseHasContentType(response, MediaType.APPLICATION_JSON,
+                    MediaType.APPLICATION_LD_PLUS_JSON));
+        }
+
+        @Override
+        public final void testFullAccessResponseAllOrNothingAuthorized() throws IOException, InterruptedException {
+            final HttpResponse<String> response =
+                    sendImageInfoRequest(ALL_OR_NOTHING_ACCESS_IMAGE, SINAI_ACCESS_TOKEN, 2);
+            final String expectedResponse =
+                    getExpectedImageInfo(ALL_OR_NOTHING_ACCESS_IMAGE, FULL_ACCESS_RESPONSE_TEMPLATE_V2, 2);
+
+            Assert.assertEquals(HTTP.OK, response.statusCode());
```

Comment:
I think I was just mirroring the use of TestUtils when I introduced these assertions a few PRs ago

Reply:
I think I (was it me?) didn't use static TestUtils because it was a class we've introduced vs. Assert which our testing library introduces. We have a pretty established history of using the static methods for Assert (across our other projects). It's not a big deal to me, but more of a consistency question. Maybe TestUtils should be using static methods too? Do you have an opinion on this?

Reply:
The distinction is somewhat arbitrary.

Reply:
ok, sure, I'm fine with that, I think I was just answering the consistency question differently. Will change

Reply:
consistency w.r.t other projects seems like a good thing

Reply:
Do you have an opinion on whether TestUtils should use static methods or not?

Reply:
I kind of like being able to see where the method lives without having to look at the top of the file (which, as you said, is less obvious for our custom classes)

Reply:
so I'd say "no"